src/ui/Pricing/SubscriptionSettings/SubscriptionCard/UserPlanCard.svelte

```diff
@@ -36,7 +36,7 @@
   onSubactionClick={isCancelled ? undefined : showCancelSubscriptionDialog}
   shouldHideBillingInfo={discount && suggestionsCount === 2}
 >
-  <p>
+  <p class="body-2">
```

Comment:
`body-2` class breaks existing desktop UI. Please, make sure it's consistent with Figma design